src/app/project/project.component.html

```diff
@@ -58,14 +58,16 @@ <h1>{{ project.name }} </h1>
             <span class="act-message ng-binding">
               <em>#</em>{{ build.id }}
             </span>
-            <span class="act-time-ago ng-binding">
-              {{ build.worker.status }}
-              <span>{{build.worker.end_time | amTimeAgo}}</span>.
-            </span>
-            <span class="act-time-duration">
-              Ran for
-              <span class="ng-binding">{{ build.worker.start_time | amDifference: build.worker.end_time :'seconds' : true | amDuration:'seconds' }}.</span>
-            </span>
+            <div *ngIf="build.worker">
```

Comment:
The API omits properties if they're empty, meaning we have to check for presence of properties throughout our code, which isn't great. I opened an issue on Brigade here about it: https://github.com/Azure/brigade/issues/385